### ISSUE TITLE
Use cypress for End to End testing of app

### ISSUE THREAD

**Issue by itaditya** [itaditya/pr-train#2]
Since the scope of the project is small instead of writing unit or integration test we can directly do E2E tests using Cypress

**Comment by mcastre** [comment]
I could help with this! Do you also want tests written or just the scaffold and setup of cypress?

**Comment by itaditya** [comment]
Yes I want the an actual test written with Cypress. Thanks for volunteering!

**Comment by mcastre** [comment]
@itaditya I noticed you referenced a PR to add cypress config to the project. Should i work off of that branch?

**Comment by itaditya** [comment]
@mcastre most of what's in that PR are the files which cypress automatically generates. So you start from scratch.
